Makefile

```diff
@@ -38,4 +39,26 @@ build:
 
 .PHONY: container
 container:
-	$(IMAGE_BUILD_CMD) build --platform=${OS}/${ARCH} -t ${IMG} .
\ No newline at end of file
+	$(IMAGE_BUILD_CMD) build --platform=${OS}/${ARCH} -t ${IMG} .
+
+tag_and_push:
+	$(IMAGE_BUILD_CMD) tag ${IMG} ${IMAGE_REPO}:${LIVE_TAG}
+	$(IMAGE_BUILD_CMD) push ${IMG} ${IMAGE_REPO}:${LIVE_TAG}
+
+fmt:
+	go fmt ./...
+
+vet:
+	go vet ./...
+
+vendor:
+	go mod tidy && go mod vendor
+
+clean: fmt vet vendor
+
+verify:
+	hack/verify-gofmt.sh
+	hack/verify-deps.sh
+
+test: vendor
```

Comment:
why does test have a dependency on vendor?

Reply:
Meant to remove this, will update